Solutions/Cortex XDR/Analytic Rules/CortexXDR_High.yaml

```diff
@@ -0,0 +1,51 @@
+id: f96728eb-9802-4522-b715-47fb66c2ecf5
+name: Cortex XDR Incident - High
+description: |
+  'A new incident was created in the Cortex XDR portal with a severity "High". Click on the events for incident details. '
+severity: High
+requiredDataConnectors:
+  - connectorId: CortexXDR
+    dataTypes:
+      - CortexXDR_Incidents_CL
+queryFrequency: 5m
+queryPeriod: 5m
+triggerOperator: gt
+triggerThreshold: 0
+suppressionDuration: 5h
+suppressionEnabled: false
+tactics: []
```

Comment:
This field is missing

Reply:
Hi @v-prasadboke, tactics have been updated for all three analytic rules as requested